### ISSUE TITLE
Site icons not shown on Macs

### ISSUE THREAD

**Issue by ospalh** [ospalh/anki-addons#57]
The site icons for the different audio source sites are not shown on Macs.
This happens only with the standard version, not when running from source
My idea is that the standard version does not bring the ico file plugin along.

**Comment by tuanmh** [comment]
Confirm, this happens on my mac as well: OSC Yosemite 10.10.2

**Comment by Zacharias030** [comment]
I have the source and other information displayed instead, relying on this terrible hack, but it's good enough for me:

In the file `review_gui.py` add the lines in between the commented lines:
```
...
if entry.icon:
    ico_label.setPixmap(QPixmap.fromImage(entry.icon))
###################################
# really bad hack
alt_text = ''
for key, value in entry.extras.items():
    alt_text += u'<br>{0}: {1}'.format(key, value)
ico_label.setText(alt_text)
###################################
```

**Comment by ospalh** [comment]
Again, OK, nice.
I don’t want to use a “terrible hack” in the pubilshed version.

**Comment by Zacharias030** [comment]
Yes, this comment was only meant for people who pass by this issue and can help themselves.
